.github/workflows/build_linux_pytorch_wheels.yml

```
@@ -43,6 +43,7 @@ jobs:
     env:
       OUTPUT_DIR: ${{ github.workspace }}/output
       PACKAGE_DIST_DIR: ${{ github.workspace }}/output/packages/dist
+      S3_PYTHON_BUCKET: ${{ inputs.s3_python_bucket }}
```

I would rename `S3_PYTHON_BUCKET` to `S3_BUCKET` as above and would further introduce an `S3_ENDPOINT` variable which makes it easier to change the URL in case needed later.

```suggestion
      S3_BUCKET: ${{ inputs.s3_bucket }}
      S3_ENDPOINT: "s3.us-east-2.amazonaws.com" 
```

---

The above `OUTPUT_DIR` seems unused. Please check and remove.